SourceryTests/Generating/StencilTemplateSpec.swift

```diff
@@ -70,12 +70,21 @@ class StencilTemplateSpec: QuickSpec {
             }
 
             describe("toArray") {
+                #if canImport(ObjectiveC)
                 context("given array") {
                     it("doesnt modify the value") {
                         let result = generate("{% for key,value in type.MyClass.variables.2.annotations %}{{ value | toArray }}{% endfor %}")
                         expect(result).to(equal("[Hello, beautiful, World]"))
                     }
                 }
+                #else
+                context("given array") {
+                    it("doesnt modify the value") {
+                        let result = generate("{% for key,value in type.MyClass.variables.2.annotations %}{{ value | toArray }}{% endfor %}")
+                        expect(result).to(equal("[\"Hello\", \"beautiful\", \"World\"]"))
```

Comment:
On linux, `stringify` from Stencil provides quotes around string values. I could not workaround it, and most likely this is due to differences in how `NSObject` works on two platforms.